WeakAuras/AuraEnvironment.lua

```diff
@@ -557,8 +557,20 @@ local exec_env_custom = setmetatable({
       return {}
     elseif overridden[k] then
       return overridden[k]
-    else
+    elseif _G[k] then
       return _G[k]
+    elseif k:match("%.") then
```

Comment:
k:find(".", 1, true) should be a bit faster, since that avoids using a lua pattern,